src/main/java/de/zalando/ep/zalenium/container/kubernetes/KubernetesContainerClient.java

```diff
@@ -610,16 +639,24 @@ public static DoneablePod createDoneablePodDefaultImpl(PodConfiguration config)
                         .endResources()
                         // Add a readiness health check so that we can know when the selenium pod is ready to accept requests
                         // so then we can initiate a registration.
+                        .withNewLivenessProbe()
+                            .withInitialDelaySeconds(30)
+                            .withTimeoutSeconds(5)
+                            .withNewHttpGet()
+                                .withPath("/wd/hub/status")
+                                .withPort(new IntOrString(40000))
+                            .endHttpGet()
+                        .endLivenessProbe()
                         .withNewReadinessProbe()
-                            .withNewExec()
-                                .addToCommand(new String[] {"/bin/sh", "-c", "http_proxy=\"\" curl -s http://`hostname -i`:"
-                                        + config.getNodePort() + "/wd/hub/status | jq .value.ready | grep true"})
-                            .endExec()
                             .withInitialDelaySeconds(5)
-                            .withFailureThreshold(60)
-                            .withPeriodSeconds(1)
-                            .withTimeoutSeconds(5)
+                            .withFailureThreshold(2)
```

Comment:
Why are there changes in the values? I'd be nice to understand the reasoning behind that.

Reply:
Hi, could you please elaborate on the previous question?